pkg/policy/repository_test.go

```diff
@@ -415,13 +415,9 @@ func (ds *PolicyTestSuite) TestMinikubeGettingStarted(c *C) {
 	expected := NewL4Policy()
 	expected.Ingress["80/TCP"] = L4Filter{
 		Port: 80, Protocol: api.ProtoTCP, U8Proto: 6,
-		Endpoints: selectorFromApp2DupList,
-		L7Parser:  "http",
-		L7RulesPerEp: L7DataMap{
-			selectorFromApp2[0]: api.L7Rules{
-				HTTP: []api.PortRuleHTTP{{Path: "/", Method: "GET"}},
-			},
-		},
+		Endpoints:        selectorFromApp2DupList,
+		L7Parser:         "",
```

Comment:
Define a constant for the empty string, along with the other constants of this type.